ph5/clients/ph5tostationxml.py

```diff
@@ -811,14 +811,13 @@ def run_ph5_to_stationxml(paths, nickname, out_format,
             p.start()
 
         results = [out_q.get() for proc in processes]
-        networks = [n for n in results if n]

```

Comment:
Thanks Derick but I think this line is required.  Otherwise there are None types in the list of networks resulting in the following errors:

`ph5tostationxml --nickname master.ph5 -p /hdf5-data2/PH5_Experiments/pn4/ --network=YW --level=network
Error - networks can only contain Network objects.`

Reply:
yeah we will have to modify that in someway. I can't see any reason that on network level requests why networks = [n for n in results if n] shoudlnt work just fine, but on every machine ive treid it on including fresh virtual machine and a new laptop this morning it doesn't>

Which is strange because I can print the network objects in results just fine.

Reply:
It's less pythonic but we could try explicitly checking if it is NoneType to see if that fixes it.

Reply:
networks = [n for n in results if n is not None]
This seems to work on first test

Reply:
how about this to make sure we only return Netowrk objects?
`networks = [n for n in results if isinstance(n, obspy.core.inventory.network.Network)]`

Reply:
I would just do the first way instead:
i.e. `networks = [n for n in results if n is not None]`

Reply:
Yeah that seems fine. There shouldn't be any way for anything except a NoneType or a network object to be in that list.